dlp/snippets/deid/deid_test.go

```diff
@@ -570,3 +570,28 @@ func TestReidTextDataWithFPE(t *testing.T) {
 		t.Errorf("reidentifyFreeTextWithFPEUsingSurrogate got %q, want %q", got, want)
 	}
 }
+
+func TestDeIdentifyTableWithCryptoHash(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	transientKey := "YOUR_TRANSIENT_CRYPTO_KEY"
+
+	if err := deIdentifyTableWithCryptoHash(&buf, tc.ProjectID, transientKey); err != nil {
+		t.Fatal(err)
+	}
+	got := buf.String()
+
+	if want := "Table after de-identification :"; !strings.Contains(got, want) {
+		t.Errorf("TestDeIdentifyTableWithCryptoHash got %q, want %q", got, want)
+	}
+	if want := `{string_value:\"my email is user3@example.org and phone is 858-555-0224\"}`; strings.Contains(got, want) {
+		t.Errorf("TestDeIdentifyTableWithCryptoHash got %q, want %q", got, want)
```

Comment:
Can you please split this to confirming email address is not present and phone number is not present.

Reply:
updated.